src/integration/HubspotApiModel.ts

```diff
@@ -41,6 +41,39 @@ export class HubspotApiModel {
         }
     }
 
+    public async createObject(objectType:string, createContactBody:object){
```

Comment:
Nice! Maybe add a brief method comment on how to user this function. Also would `createContact` be a better name? At your discretion, but that might be a bit clearer, or if you want to keep this generic, change the consts to not mention contact (eg. createObjectURL).